packages/core/__tests__/css.js

```diff
@@ -188,3 +188,32 @@ test('child selector array', () => {
 
   expect(tree.toJSON()).toMatchSnapshot()
 })
+
+test('warn about tagged template arguments weirdly forwarded', () => {
+  function media(...args) {
+    return css`
+      @media (min-width: 100px) {
+        ${css(...args, 'background-color: #fff;')};
+      }
+    `
+  }
+
+  const oldConsoleWarn = console.warn
+  // $FlowFixMe
+  console.warn = jest.fn()
+
```

Comment:
This test should use this helper. https://github.com/emotion-js/emotion/blob/master/scripts/test-utils/src/index.js#L11